cns/service/main.go

```diff
@@ -231,10 +240,10 @@ func main() {
 
 	// Start CNS.
 	if httpRestService != nil {
-		// go telemetry.SendCnsTelemetry(reportToHostInterval,
-		// 	reports,
-		// 	httpRestService.(*restserver.HTTPRestService),
-		// 	telemetryStopProcessing)
+		go telemetry.SendCnsTelemetry(reportToHostInterval,
```

Comment:
need to check if telemetry enabled before calling this

Reply:
done